### ISSUE TITLE
Does this work with cloudflare? 

### ISSUE THREAD

**Issue by ElasticBottle** [dotenvx/dotenvx#280]
Love this work!

Tried using encrypted `.env` with cloudflare but am noticing that the decrypted values isn't being inject and that cloudflare is reading the raw values directly.

Here's how I'm running it

`dotenvx run -f ./.dev.vars -- wrangler dev`

Curious to know if I'm doing something wrong!

**Comment by motdotla** [comment]
can you share a little more of what your `wrangler.toml` file looks like?

i haven't used wrangler myself. seems like it does not respect a machine's environment variables at first glance. what it calls environment variables are really just hardcoded vars in the toml file?

**Comment by motdotla** [comment]
we'll be adding a `dotenvx decrypt` command. at that time, we'll probably support piping the output to another file - which then could give you this convenience.

**Comment by motdotla** [comment]
`dotenvx decrypt` is now available in 1.6.0 and greater.

```
$ dotenvx decrypt
$ dotenvx decrypt --stdout
$ dotenvx decrypt --stdout > somefile.txt
```
